### ISSUE TITLE
Popup doesn't navigate after complete of deferral

### ISSUE THREAD

**Issue by ukandrewc** [MicrosoftEdge/WebView2Feedback#680]
**Description**
When using NewWindowRequested with e.GetDeferral, after setting e.NewWindow and calling Complete, the page is either unititialized or blank.

**Version**
SDK: 1.0.707-prerelease
Runtime: 88.0.702.0 (Official build) dev (32-bit)
Framework: WinForms
OS: Win7

**Repro Steps**

```
Private Sub CoreWeb_NewWindowRequested(sender As Object, e As CoreWebView2NewWindowRequestedEventArgs) Handles CoreWeb.NewWindowRequested
	Dim Browser As New WVBrowser ' Inherited WebView2
	Browser.Deferral = e.GetDeferral
	Browser.EventArgs = e
End Sub

Private Sub WVBrowser_CoreWebView2InitializationCompleted(sender As Object, e As CoreWebView2InitializationCompletedEventArgs) Handles Me.CoreWebView2InitializationCompleted

	If Deferral IsNot Nothing Then
		EventArgs.NewWindow = CoreWebView2
		Deferral.Complete()
		Deferral = Nothing
		EventArgs = Nothing
	End If
End Sub

```
**Screenshots**
### WebView2:
![image](https://user-images.githubusercontent.com/30246320/100316011-97f40280-2fb1-11eb-86f0-942e50c98e55.png)

### Edge:
![image](https://user-images.githubusercontent.com/30246320/100315976-83176f00-2fb1-11eb-8e07-4efd1d5d8a87.png)

[AB#30943865](https://microsoft.visualstudio.com/90b2a23c-cab8-4e7c-90e7-a977f32c1f5d/_workitems/edit/30943865)

**Comment by champnic** [comment]
Thanks for the report! I've added this bug to our backlog.

**Comment by schuylergleaves** [comment]
Hey @ukandrewc, was this only occurring for popups with target=_blank?

**Comment by ukandrewc** [comment]
@schuylergleaves That is likely, I will check.

**Comment by schuylergleaves** [comment]
If so, then I believe it is the same issue and the next runtime update should solve this, sorry for the delay.

**Comment by schuylergleaves** [comment]
@ukandrewc Did you ever get a chance to verify that this was occurring for popups only with target=_blank? This appears to be the same issue as #881 and is fixed (will ship in runtime 93.0.924.0+). If you are not receiving this for any popups without target=_blank, I am going to go ahead and mark this as closed 😄

**Comment by ukandrewc** [comment]
@schuylergleaves Sorry I didn't get a chance. We're stalled on our WebView2 dev at the moment, so it hasn't had priority.
I'll check #881 and come back if it's not the same. Thanks.
